package.json

```diff
@@ -32,12 +32,13 @@
     "eslint": "^9.14.0",
     "globals": "^15.12.0",
     "lodash": "^4.17.21",
+    "looks-same": "^9.0.1",
```

Comment:
for what we use that package and what it can do, we should probably write the looks-same for colors ourselves, but I'll let you do what you want.